dashboard/app/controllers/lti/v1/lti_sections_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Lti
+  module V1
+    class LtiSectionsController < ApplicationController
```

Comment:
It's already within the `Lti` namespace
```suggestion
    class SectionsController < ApplicationController
```